controllers/controllers/cluster_controller_test_test.go

```diff
@@ -0,0 +1,112 @@
+package controllers_test
```

Comment:
I created this file to have tests in `controllers_test` package
The rest of the current tests in `controllers/controllers/cluster_controller_test.go` should we move to this package if possible.